scripts/dev/dev_config.py

```diff
@@ -43,15 +43,18 @@ def testrunner_image(self):
         return self._config["testrunner_image"]
 
 
-def load_config(config_file_path: str = None) -> Optional[DevConfig]:
+
+def load_config(config_file_path: str = None) -> DevConfig:
     if config_file_path is None:
         config_file_path = get_config_path()
-    with open(config_file_path, "r") as f:
-        return DevConfig(json.loads(f.read()))
 
-    print(
-        "No DevConfig found. Please ensure that the configuration file exists at '{}'".format(
-            config_file_path
+    try:
+        with open(config_file_path, "r") as f:
+            return DevConfig(json.loads(f.read()))
+    except FileNotFoundError:
+        print(
+            "No DevConfig found. Please ensure that the configuration file exists at '{}'".format(config_file_path)
         )
-    )
-    return None
+        raise
+    except Exception:
+        raise
```

Comment:
just seeing now, we don't need to actually need this generic catch all if we just want to  re-throw this exception, this will happen anyway if it is not of type `FileNotFoundError`